Algebraic_kernel_d/examples/Algebraic_kernel_d/CMakeLists.txt

```diff
@@ -18,20 +18,17 @@ endif()
 include( ${CGAL_USE_FILE} )
 
 
-find_package( Boost REQUIRED )
+# Boost and its components
+find_package( Boost QUIET )
 if ( NOT Boost_FOUND )
   message(STATUS "This project requires the Boost library, and will not be compiled.")
-  return()  
+  return()
 endif()
 
-
-find_package( MPFI REQUIRED )
-if ( NOT MPFI_FOUND )
-  message(STATUS "This project requires the MPFI library, and will not be compiled.")
-  return()  
+find_package( MPFI QUIET )
```

Comment:
I see. It partially works - e.g., it is certainly useful if paths to 3rd-party libs in non-standard places can be preconfigured once and for all, and the generic CMakeLists.txt with a default CMake invocation just works.
It is certainly incomplete, e.g., not all environment variables in the build process are translated to the proper CMake variables (e.g., RS{,3}_{INC,LIB}_DIR are not acknowledged for some reason, and I already bugged Eric about that while he was still around).

But if noone understands the scripts anymore, that's more or less academic - if noone can maintain it, it's doomed anyway.